### DISCUSSION TITLE
Update an app then rollback to non-updated version ?

### DISCUSSION THREAD

**Discussion by blockchainexpert2022** [Tyrrrz/Onova#57] [General]
Hi, and thank you for your really good work.
I'd like to know if it is possible to update an app then run the updated version then rollback to the non updated version before quitting the app or just after the app has been exited ?
Thanks !

**Comment by Tyrrrz** [comment]
Hi.

What you could do is update to the previous version. If you know the exact version, you can just prepare the corresponding update and install it.
